timer/timer.ino

```diff
@@ -77,9 +77,12 @@ void getMachineInput() {
 
   if (millis() - serialUpdateMillis > 5000) {
     serialUpdateMillis = millis();
-    memset(receivedChars, 0, numChars );
-    Serial.println("Request serial update");
-    mySerial.write(0x11);
+    // Looking for a known bad character from the Mara X and skipping the update if present
+    if (String(receivedChars[0]) != "⸮") {
+      memset(receivedChars, 0, numChars );
+      Serial.println("Request serial update");
+      mySerial.write(0x11);
```

Comment:
This serial write tells MaraX to send another stream of data.
If the first character is "⸮" it will never tell MaraX to send new data

Reply:
maybe in the else statement of
` if (rc != endMarker) ` (basically if the received character is the end marker) chec if the string contains a "⸮" and if so, discard the string.

But what this will do, will just ignore the data, and you will never know you have a problem, right? maybe users should see these strange characters and investigate and eventually change the brick